src/foundry/client/pixi/sources/light-source.d.ts

```diff
@@ -271,143 +275,245 @@ declare global {
      */
     protected _initializeBlending(): void;
 
+    /** @override */
```

Comment:
There are still some `@override` in here.

Reply:
Updated